<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>184</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0" colspan="3">
      <widget class="QRadioButton" name="orientationRadioButton">
       <property name="text">
        <string>Orientation</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="targetNumSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>359</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QRadioButton" name="positionRadioButton">
       <property name="text">
        <string>Position</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="seqNumSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>386</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="targetNumLabel">
       <property name="text">
        <string>Target #</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="seqNumLabel">
       <property name="text">
        <string>Seq #</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QPushButton" name="plotTrajectoryButton">
       <property name="text">
        <string>Plot</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="subjectNumSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="subjectNumLabel">
       <property name="text">
        <string>Subject #</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QPushButton" name="clearPlotsButton">
       <property name="text">
        <string>Clear Plots</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>131</width>
      <height>83</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QRadioButton" name="fixedSourceRadioButton">
       <property name="text">
        <string>Fixed Source</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">fixedPointGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="fixedTargetRadioButton">
       <property name="text">
        <string>Fixed Target</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">fixedPointGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="noneFixedRadioButton">
       <property name="text">
        <string>None</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">fixedPointGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>290</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="xLabel">
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="yEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>0.0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="yLabel">
       <property name="text">
        <string>y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="xEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>0.0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="orientationLabel">
       <property name="text">
        <string>orientation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="orientationEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>0.0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="fixedPointGroup"/>
 </buttongroups>
</ui>
